### ISSUE TITLE
Update issue

### ISSUE THREAD

**Issue by mischugo** [utom/sketch-measure#333]
Hi,

the new version of Sketch app has it's own plug in updater (inidcates in the upper right corner, if any update is available)

Using [Sketch-Plugin-Manager](https://github.com/mludowise/Sketch-Plugin-Manager) will not update the plugin in then. Have a look to the plug in folder. There are two folders "sketch-measure", one with the plug in inside, one without ...

**Comment by utom** [comment]
Yep, I see that.  Any updates isn't working via the Sketch Updater. Try to download new version from web.

**Comment by luxlogica** [comment]
@utom It's likely that this issue can be safely closed now.
